event-management-backend/src/main/resources/application.properties

```diff
@@ -7,6 +7,6 @@ event-management.jwt.aws.region=eu-west-1
 server.error.include-message=always
 event-management.s3.tickets.bucketName=event-management-tickets
 
-spring.profiles.active=prod
+spring.profiles.active=${PROFILE:default}
```

Comment:
That's an excellent point. Could we also use this approach for connecting to the database?

Reply:
Changing the profile already changed the db that was in use. I added env. variables for the db connection on top if this.

Reply:
Excellent.

The issue was that the connection credentials were still baked into the application. If we change the DB, we have to rebuild the app. If we deploy it to multiple environments, we need multiple versions.